src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -2390,6 +2390,24 @@ private function registerNotifierConfiguration(array $config, ContainerBuilder $
             $container->removeDefinition('notifier.channel.email');
         }
 
+        $servicesWithBusArgument = ['texter', 'chatter', 'notifier.channel.chat', 'notifier.channel.email', 'notifier.channel.sms'];
+        if (false === $messageBus = $config['message_bus']) {
+            foreach ($servicesWithBusArgument as $serviceId) {
+                if ($container->hasDefinition($serviceId)) {
+                    $container->getDefinition($serviceId)->replaceArgument(1, null);
```

Comment:
shouldn't it be the `messenger.default_bus` service? like previous implementation

Reply:
This case handle the situation where I don't want to have a message bus at all.